baybe/acquisition/base.py

```diff
@@ -39,15 +39,17 @@ def to_botorch(
         self,
         surrogate: Surrogate,
         searchspace: SearchSpace,
-        train_x: pd.DataFrame,
-        train_y: pd.DataFrame,
+        measurements: pd.DataFrame,
     ):
         """Create the botorch-ready representation of the function."""
```

Comment:
Agree, even though this also was the case in the previous version with `train_x`/`train_y`. In order to avoid having to write a full doc that would either contain copy-pasted content for `surrogate`/`searchspace` or references to other functions (while for these argument the context is automatically clear through their annotations), would it suffice if I just write the following?

```python
        """Create the botorch-ready representation of the function.

        The required structure of `measurements` is specified in
        :meth:`babye.recommenders.base.RecommenderProtocol`.
        """
```